VogtObserver3/Store.cs

```diff
@@ -18,13 +20,21 @@ public void Notify()
         {
             foreach (var observer in _observers)
             {
-
+                observer.Update(_discount);
             }
         }
 
         public void RemoveObserver(Observer o)
         {
             _observers.Remove(o);
         }
+
+
+        public void GetDiscount()
```

Comment:
I would call this applyDiscount

Reply:
This must be a storewide discount. Because the discount is on the store, not on something called Product or Item, etc.

Reply:
Looking at the chegg requirments, it looks like ur supposed to update the customer's discount state? So you should be passing in the _discount TO the notify() function `Notify(_discount)` and keeping track ofthe customer's discount within the customer object.

The purpose is prob so u can later just check the _discoutn on the customer when the CUSTOMER is making purchases.